src/screens/MemberDetail/MemberDetail.spec.tsx

```
@@ -0,0 +1,148 @@
+      action,
+      isActive,
+    }: {
+      title: string;
+      action: () => void;
+      isActive: boolean;
+    }) => (
+      <button
+        data-testid={`tab-${title}`}
+        data-active={isActive}
+        onClick={action}
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)